data/translations/ja-JP.yaml

```diff
@@ -25,10 +25,10 @@ topsectioncus: お問い合わせ
 topsectionwhy: なぜイソリアか？
 topsectionwwd: 主なソリューション
 topsectionwwdtext: イソリアのプロフェショナルスタッフはインフラから、オペレーション、開発、マネージメントとガバナンスまで幅広い範囲にわたるIT活動に対応いたします。貴社の指令、規格と委任に基づいて、貴社が必要とされるニーズに必ずお応えいたします。
-topslogan: イソリアにお任せ下さい
-topsubslogan: 貴社の頼れるIT部となります
-toptestimoninal1: イソリアが我々の複雑で、複数年に及ぶプロジェクトの管理を成功に導いた時、まるで長いトンネルの先に希望の光が差し込んだような状況でした。
-toptestimoninal2: イソリアの技術的な知識は誰にも劣らないうえに、誰でも詳細が理解できるように一般的な用語で複雑な解決方案を説明いただきました。
+topslogan:　イソリアにお任せください！
```

Comment:
Oh, no… sorry about this.



*From:* Rick Cogley [mailto:notifications@github.com]
*Sent:* Tuesday, June 09, 2015 3:31 PM
*To:* eSolia/eSolia
*Cc:* eishikawa
*Subject:* Re: [eSolia] Ei translation 20150609 (#37)



In data/translations/ja-JP.yaml
<https://github.com/eSolia/eSolia/pull/37#discussion_r31985552>:

> @@ -25,10 +25,10 @@ topsectioncus: お問い合わせ

>  topsectionwhy: なぜイソリアか？

>  topsectionwwd: 主なソリューション

>  topsectionwwdtext: イソリアのプロフェショナルスタッフはインフラから、オペレーション、開発、マネージメントとガバナンスまで幅広い範囲にわたるIT活動に対応いたします。貴社の指令、規格と委任に基づいて、貴社が必要とされるニーズに必ずお応えいたします。

> -topslogan: イソリアにお任せ下さい

> -topsubslogan: 貴社の頼れるIT部となります

> -toptestimoninal1: イソリアが我々の複雑で、複数年に及ぶプロジェクトの管理を成功に導いた時、まるで長いトンネルの先に希望の光が差し込んだような状況でした。

> -toptestimoninal2: イソリアの技術的な知識は誰にも劣らないうえに、誰でも詳細が理解できるように一般的な用語で複雑な解決方案を説明いただきました。

> +topslogan:　イソリアにお任せください！

The Japanese space between : and the text caused the site to break
@eishikawa <https://github.com/eishikawa>

—
Reply to this email directly or view it on GitHub
<https://github.com/eSolia/eSolia/pull/37/files#r31985552>.